mods/tnt/init.lua

```diff
@@ -1,10 +1,10 @@
 tnt = {}
 -- Default to enabled in singleplayer and disabled in multiplayer
 local singleplayer = minetest.is_singleplayer()
-local setting = minetest.setting_getbool("enable_tnt")
-if (not singleplayer and setting ~= true) or
-		(singleplayer and setting == false) then
-	return
+local enable_tnt = minetest.setting_getbool("enable_tnt")
+if (not singleplayer and not enable_tnt) or
```

Comment:
getbool can return 'nil' so this line should not be changed, as it was checking for setting == false or setting == nil